add: Youtube playlist at waf rule sets doc (#1224)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx

---

import LinkButton from 'azion-webkit/linkbutton'

> import Tabs from '~/components/tabs/Tabs'

Reviewer: Delete if you aren't using it on the file.

| Unwanted Access | Detects attempts to access administrative or vulnerable pages, bots, and security scanning tools |
| Identified Attack | Prevents several types of common attacks and known vulnerabilities that should certainly be blocked |

> Discover how to protect your application from attacks like XSS, SQL Injection, RFI, and Directory Traversal with our exclusive playlist! Check it out below:

Reviewer: ```suggestion
Discover how to protect your application from attacks like XSS, SQL Injection, RFI, and Directory Traversal with this playlist! Check it out below:
```

<div class="cms-embed">

>   <iframe width="600" height="400" src="https://www.youtube.com/embed/VIDEO_ID?playlist=nvtQ00A6An4,b2H3cPDPKhk,s-Pr5Xg8qFg,Z3kuwg-lmb4" loading="lazy" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

Reviewer: Review if title must be customize.
